pkgs/development/tools/analysis/codeql/default.nix

```diff
@@ -1,26 +1,18 @@
-{ lib, stdenv
-, fetchzip
-, zlib
-, xorg
-, freetype
-, alsa-lib
-, jdk11
-, curl
-, lttng-ust
-, autoPatchelfHook
-}:
+{ lib, stdenv, fetchzip, zlib, xorg, freetype, alsa-lib, jdk11, curl, lttng-ust
+, autoPatchelfHook }:
 
 stdenv.mkDerivation rec {
   pname = "codeql";
-  version = "2.5.9";
+  version = "2.6.0";
 
   dontConfigure = true;
   dontBuild = true;
   dontStrip = true;
 
   src = fetchzip {
-    url = "https://github.com/github/codeql-cli-binaries/releases/download/v${version}/codeql.zip";
-    sha256 = "sha256-r3Jm+VYjn0Dz4BCSbADbgTWL1owbyIXlkoj6mOmZcZk=";
+    url =
+      "https://github.com/github/codeql-cli-binaries/releases/download/v${version}/codeql.zip";
```

Comment:
```suggestion
    url = "https://github.com/github/codeql-cli-binaries/releases/download/v${version}/codeql.zip";
```

Reply:
I've run this file through both `nixfmt` and `nixpkgs-fmt` and both produce this output. Are you sure we want to change it?

Reply:
Yes, I am sure. nixfmt has an old fashioned idea about line lengths and nixpkgs-fmt does not reverse that decission.